sdk.yaml

```diff
@@ -286,6 +286,13 @@ Realtime:
       .synopsis: |
         Send a message on this channel.
         This is a message over the Realtime connection, not a REST operation.
+    State Events:
+      .specification:
+        [RTL2, TH1, TH2, TH3, TH4, TH5, RTN11d, RTL4e, RTL3a, RTL4g, RTL14, RTL24]
+      .synopsis: |
+        `ChannelEvent` events can be received asynchronously when channel state changes.
+        The current `ChannelState` can be synchronously queried (e.g. via a `state` property on the channel instance).
+        Also provides access to an error reason object in case of failure.
```

Comment:
Generally, any transition, not just failure.

Reply:
Thanks. I'll look at this sentence again.

Reply:
Addressed in https://github.com/ably/features/pull/34/commits/c23fc2151c354073061fc454cde0ccadff73b83f.

Reply:
I'm resolving this conversation on @paddybyers' behalf to enable me to get this pull request merged cleanly.